frontend/src/Admin/queries/portfolioPeriods.graphql

```diff
@@ -0,0 +1,17 @@
+query PortfolioPeriods {
+  portfolioPeriods(orderBy: { sort: "entryStart", direction: DESC }) {
+    id
+    name
+    description
+    numPieces
+    entryStart
+    entryEnd
+    judgingStart
+    judgingEnd
+    createdAt
+    updatedAt
+    judges {
+      username
```

Comment:
Even though this is here in the query, the Judges aren't actually attached in the db in any way whatsoever.

Reply:
Do we have a ticket to do that later on?

Reply:
We don't, Adding one now.